holidays/countries/thailand.py

```diff
@@ -459,30 +477,59 @@ def _add_with_observed(dt, holiday_name) -> None:
         # วันคล้ายวันสวรรคตพระบาทสมเด็จพระปรมินทร มหาภูมิพลอดุลยเดช บรมนาถบพิตร
         # Status: In-Use
         # Started in 2017 (B.E 2560)
+        # Got conferred with 'the Great' title in 2019 (B.E. 2562)
 
-        if year >= 2017:
+        if 2017 <= year <= 2018:
+            _add_with_observed(
+                date(year, OCT, 13),
+                self.tr(
+                    "วันคล้ายวันสวรรคตพระบาทสมเด็จพระปรมินทรมหาภูมิพลอดุลยเดช บรมนาถบพิตร"  # noqa: E501
+                ),
```

Comment:
This technique allows to avoid E501 suppression.
```suggestion
                self.tr(
                    "วันคล้ายวันสวรรคตพระบาทสมเด็จพระปรมินทรมหาภูมิพลอดุลยเดช "
                    "บรมนาถบพิตร"
                ),
```